docs/consumer/Receiver.md

```diff
@@ -1,6 +1,81 @@
 # Stake Receiver
 
 The Stake Receiver is on the consumer side and connected an External Staker on the Provider side.
-This handles the normalization of the token streams and communicates with the Meta-Staker.
+This handles the normalization of the token streams and communicates with "Virtual Staking".
+It is connected to the Provider chain via IBC and "receives" the various staking commands
+
+**TODO** This was called the "Consumer contract" in previous versions and probably needs a better name
+
+## Setup
+
+When we [deploy the contracts](../ibc/Overview.md#deployment), we connect the Stake Receiver on the consumer chain
+with an [External Staking](../provider/ExternalStaking.md) contract on the Provider. Once this connection is established,
+Consumer governance can authorize this Stake Receiver with some ability to mint on the ["Virtual Staking" contract](./VirtualStaking.md).
+
+Note that when we deploy the Stake Receiver, we configure the address of the "virtual staking" contract. We must also
+define a price oracle contract on setup. (see ["Price Normalization"](#price-normalization) below)
```

Comment:
So, I think the price oracle could be _optional_. If you're operating a 100% consumer chain the price oracle is not needed so much, additionally governance may be sufficient enough for adjusting price and discount rates in some cases.

Anyway, we can create an interface for updating prices that could be used by governance or something like an oracle.

Reply:
Good point.
I will make a section on oracles and show an interface and different options

Reply:
Expanded on this in 2016663